brave/brave-core
# Add options for enable/disable autocomplete in omnibox

When disabled, OmniboxController::StartAutocomplete() does early
return instead of requesting to AutocompleteController.

This option is disabled by default.

fix https://github.com/brave/brave-browser/issues/843
![image](https://user-images.githubusercontent.com/6786187/69383581-6214a200-0cfd-11ea-80a2-a311e9647dfb.png)

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
`yarn test brave_browser_tests --filter=OmniboxAutocompleteTest*`

Manual test steps
1. Launch browser and load settings page
2. Check disable autocomplete option is turned off by default
3. Typing in omnibox and autocomplete works
4. Toggle autocomplete option 
5. Typing in omnibox and autocomplete doesn't work.(popup is not shown and inline autocomplete doesn't work)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
